app/models/manageiq/providers/redhat/infra_manager/provision/configuration.rb

```diff
@@ -29,7 +29,7 @@ def configure_sysprep
   end
 
   def configure_container
-    vm.with_provider_object(:version => vm.ext_management_system.highest_allowed_api_version) do |rhevm_vm|
+    vm.with_provider_object(:version => ManageIQ::Providers::Redhat::InfraManager::ApiIntegration::SUPPORTED_API_VERSION) do |rhevm_vm|
```

Comment:
Do we still need this?
Maybe with_provider_object does not require a version at all anymore?

Reply:
You're right, I removed version parameter from all the other with_provider_object and with_provider_connection but I left this one. I'll remove it